e2e/test/scenarios/dashboard/dashboard-reproductions.cy.spec.js

```diff
@@ -1500,3 +1504,46 @@ describe("issue 42165", () => {
     });
   });
 });
+
+describe("issue 47170", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.request("POST", `/api/bookmark/dashboard/${ORDERS_DASHBOARD_ID}`);
+
+    createDashboard({ name: "Dashboard A" }, { wrapId: true }).then(
+      dashboardId => {
+        cy.request("POST", `/api/bookmark/dashboard/${dashboardId}`);
+      },
+    );
+
+    cy.intercept(
+      {
+        method: "GET",
+        url: "/api/dashboard/*",
+        middleware: true,
+      },
+      req => {
+        req.on("response", res => {
+          res.setThrottle(2);
+        });
```

Comment:
:x:  In order for the issue to be reproduced, the dashboard request needs to get cancelled. This does not currently happen.

It looks like setThrottle does not help much for empty dashboards that are < 1 KB - the requests still finish in less than a second.

This does the trick (we probably need delay > `1000` to make it reliable in CI):

```suggestion
        req.continue(res => {
          return new Promise(resolve => setTimeout(resolve, 1000));
        });
```